modules/installation-guide/partials/assembly_upgrading-che-using-operatorhub.adoc

```diff
@@ -0,0 +1,20 @@
+// Module included in the following assemblies:
+//
+// upgrading-{prod-id-short}
+
+[id="upgrading-{prod-id-short}-using-operatorhub_{context}"]
+= Upgrading {prod-short} using OperatorHub
+
+This section describes how to upgrade from an earlier minor version using the Operator from OperatorHub in the OpenShift web console.
+
+OperatorHub supports `Automatic` and `Manual` upgrade strategies. 
+If the approval strategy in the subscription is set to `Automatic`, the upgrade process starts when a new version of the Operator is published.
+If the approval strategy in the subscription is set to `Manual`, the update must be manually approved every time the new version of the Operator is published.
```

Comment:
Maybe a table here would increase readability?

Reply:
Could we clarify how the manual approval happens ? screenshot could help